### PR TITLE
Added  zookeeper 3.6.0

### PR DESCRIPTION
**Pull request by jaceq**
Added newly release zookeeper 3.6.0.
I did some local testing.

### CODE REVIEW

File: 3.6.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,84 @@
+FROM openjdk:8-jre-slim
```
Comment (amuraru) [inline 3.6.0/Dockerfile]: let's use jdk:11 - zk 3.6 works on java11

Comment (jaceq) [reply 3.6.0/Dockerfile]: Done, I build it locally and it runs on jdk 11

**Comment by 31z4** [comment]
Thanks for your contribution @jaceq 👍

**Comment by 31z4** [comment]
https://github.com/docker-library/official-images/pull/7624

**Pull request merged**